### PR TITLE
[App Switch] Update CHANGELOG

### PR DESCRIPTION
**Pull request by richherrera**
### Summary of changes

 - Update CHANGELOG

### Checklist

 - [x] Added a changelog entry
 - [ ] Relevant test coverage

### Authors

@richherrera

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
-
+* PayPal
+  * Add PayPal App Switch flow (BETA)
```
Comment (jaxdesmarais) [inline CHANGELOG.md:5]: Can we add a couple notes on how to use the feature here as well? We can use iOS as a reference: https://github.com/braintree/braintree_ios/blob/main/CHANGELOG.md#6210-2024-06-12

Comment (richherrera) [reply CHANGELOG.md]: Sure, updated: [4914007](https://github.com/braintree/braintree_android/pull/1166/commits/4914007efc5333ccf4927bb281b2ac96664a92f5) let me know if this looks good

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
-
+* PayPal
+  * Add PayPal App Switch vault flow (BETA)
+    * Add `enablePayPalAppSwitch` property to `PayPalVaultRequest` for App Switch support
+    * Require `PayPalVaultRequest.userAuthenticationEmail` for App Switch support
+    * Require `PayPalClient.appLinkReturnUrl` for App Switch support
+    * Send `link_type` and `paypal_installed` in `event_params` when available to PayPal's analytics service (FPTI)
```
Comment (tdchow) [inline CHANGELOG.md:9]: Is this something we can omit from the changelog? I think this would be an internal change.

Comment (richherrera) [reply CHANGELOG.md:9]: yeah 🤔 , im basically following the same approach we used for [iOS](https://github.com/braintree/braintree_ios/pull/1335/commits/ec3488a72c332087048470f408268314077d78dc). In fact, we’ve even updated the CHANGELOG with internal changes in the past. I’m open to making adjustments if we agree as a team on omitting these types of changes.

Comment (tdchow) [reply CHANGELOG.md:9]: Gotcha! we can merge this in as is and modify it if the team decides to only include external changes.

**Review by tdchow — APPROVED** [review]

**Review by jaxdesmarais — APPROVED** [review]

**Pull request merged**